web/css/doc.css

```diff
@@ -128,3 +134,43 @@ body {
 #help .subs.instances {
     display: none;
 }
+
+h3.ui-accordion-header {
+    color: var(--doc-accordion-color);
+    background: var(--doc-accordion-background);
+}
+
+h3.ui-accordion-header-active {
+    color: var(--doc-expanded-accordion-color);
+    background: var(--doc-expanded-accordion-background);
+}
+
+div.ui-accordion-content {
+    background: var(--doc-accordion-background);
+}
+
+.ui-widget-content a {
+    color: var(--doc-accordion-color);
+}
+
+.md svg.diagram {
+    fill: var(--doc-svg-fill) !important;
+    stroke: var(--doc-svg-stroke) !important;
+}
+
+/* Haskell-mode. */
```

Comment:
I'm not sure what this comment means, but nothing here is specific to haskell mode.  Wouldn't it also apply to the reference sections of codeworld-mode?

Reply:
Deleted.